services/QuillLMS/app/queries/admin_diagnostic_reports/diagnostic_aggregate_query.rb

```diff
@@ -0,0 +1,150 @@
+# frozen_string_literal: true
+
+module AdminDiagnosticReports
+  class DiagnosticAggregateQuery < Snapshots::PeriodQuery
+    class InvalidAggregationError < StandardError; end
+
+    attr_reader :additional_aggregation
+
+    AGGREGATION_OPTIONS = [
+      'grade',
+      'teacher',
+      'classroom'
+    ]
+    DIAGNOSTIC_ORDER_BY_ID = [
+      1663, # Starter Baseline Diagnostic (Pre)
+      1668, # Intermediate Baseline Diagnostic (Pre)
+      1678, # Advanced Baseline Diagnostic (Pre)
+      1161, # ELL Starter Baseline Diagnostic (Pre)
+      1568, # ELL Intermediate Baseline Diagnostic (Pre)
+      1590  # ELL Advanced Baseline Diagnostic (Pre)
+    ]
```

Comment:
I'm not super-happy with hard-coding the IDs in here, and there are methods for identifying active Pre Diagnostics without doing this, but because we have a specific order that we want to maintain for these queries, this was the cleanest thing I could come up with.  I'd be totally open to alternate approaches that involve less risk around forgetting to update these and not being able to figure out why a new Diagnostic isn't loading in this report.